lib/fastlane/plugin/create_remote_config_release_package/actions/create_remote_config_release_package_action.rb

```diff
@@ -163,6 +165,16 @@ def self.available_options
             verify_block: proc do |value|
               UI.user_error!("No Xcode project path given, pass using `project_file_path: 'project_file_path'`") unless value && !value.empty?
             end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :target_name,
+            env_name: "RC_TARGET",
+            description: "specifies the target",
+            optional: false,
```

Comment:
This can be an optional parameter, and only needed when a project has multiple targets.